src/util.py

```diff
@@ -260,12 +265,86 @@ def xr_add_cyclic_longitudes(da, coord):
 
     return new_da
 
+
+def set_vector_density(data, minDistance=0, otherVars=[]):
+    """
+    Utility function to change density of vector plots.
+
+    Ars:
+
+        data (:class:`xarray.core.dataarray.DataArray`):
+            Data array that contains the vector plot latitude/longitude data.
+
+        minDistance (:class:`int`):
+            Value in degrees that determines the distance between the vectors.
+
+        otherVars (:class: array of `xarray.core.dataarray.DataArray`):
+            Other variables that require the same density reduction as the lat/lon data
+
+    """
+    import math
+    import warnings
+
+    if minDistance <= 0:
+        raise Exception("minDistance cannot be negative or zero.")
+    else:
+        lat_every = 1
+        lon_every = 1
+
+        # Get distance between points in latitude (y axis)
+        lat = data['lat']
+        latdifference = (float)(lat[1] - lat[0])
+
+        # Get distance between points in longitude (x axis)
+        lon = data['lon']
+        londifference = (float)(lon[1] - lon[0])
+
+        # Get distance between points that are diagonally adjacent
+        diagDifference = math.sqrt(latdifference**2 + londifference**2)
+
+        # Initialize ds
+        ds = data
+
+        if diagDifference >= minDistance and latdifference >= minDistance and londifference >= minDistance:
+            warnings.warn('Plot spacing is alrady greater or equal to ' + (str)(minDistance))
+
+        # While diagD
```

Comment:
This comment should be descriptive to explain what is being done inside the while-loop. For example, I had to read the entire while-loop to understand by myself.